### ISSUE TITLE
Can I use SwampDragon in Production 

### ISSUE THREAD

**Issue by a1Gupta** [jonashagstedt/swampdragon#168]
Hi, Is it safe to use the existing SwampDragon in production. I am aware of the fact that its dependent on tornado-redis which is no longer maintained and @jonashagstedt is working on removing this as well as django's dependency.

**Comment by jonashagstedt** [comment]
That's kind of an impossible question to answer.
It's not safe to use anything in production.

Could you be more specific?

**Comment by a1Gupta** [comment]
I wanted clarity specifically regarding these  -
1. Is current version of SwampDragon stable and compatible with Django 1.7 and Django 1.8 ?
2. Are you planning to do major changes in SwampDragon ?
3. Is current version of tornado-redis stable and compatible with Django 1.7 and Django 1.8 ?

I have read your reply to Issue #161 and that was helpful. I just wanted to ensure that the new version should be adaptable with minimal overhead and the current version is stable for production with Django 1.7 and Django 1.8

**Comment by jonashagstedt** [comment]
The plan is to try to keep the changes under the hood, so implementation vise there should be as few changes as possible.

If changes will be required before upgrading they should (ideally) be minimal.
However the undertaking of factoring out some of these things will take some time so I would not stress about the upgrade at this point.

**Comment by a1Gupta** [comment]
That would be good :) I understand removing dependency and bringing support for other python frameworks will take time. Till then can I use this with Django 1.7 and Django 1.8 ?

**Comment by jonashagstedt** [comment]
Yes it shouldn't be an issue using it with Django 1.7 or 1.8 (if you run into any issues you should be able to install it from Github, latest master)

**Comment by a1Gupta** [comment]
Yeah that's possible. Thanks @jonashagstedt  All the best! Hope to see the new SwampDragon in few months :)

**Comment by a1Gupta** [comment]
Any updates regarding issue #135 ? Feeling Stuck
